### ISSUE TITLE
Think about renaming field is_valid of Status

### ISSUE THREAD

**Issue by vsoch** [yeastphenome/yeastphenome.org#122]
Per discussion here: https://github.com/yeastphenome/yeastphenome.org/pull/121#pullrequestreview-555916039

> This field has now become a misnomer. is_valid=True was (so far) used to indicate statuses that correspond to a situation where we already have or expect to have data for a paper (e.g., "loaded", "to request", "request sent" etc.). In contrast is_valid=False corresponds to statuses where, for various reasons, the data will not be available (e.g., "not available", "request abandoned", "not relevant").
"Not relevant" here is an outlier -- it indicates a paper that exists in the DB, but should always be ignored because it is outside of our project's scope and we are only keeping it for our records. In contrast, "not available" means that the paper is relevant, we should have gotten data, but the authors confirmed that the data has been lost forever. "Request abandoned" means that we failed to get a response from the authors and, as far as we're concerned, the data is lost forever.
So, long story short -- at some point we should rename is_valid to something else. But for now, it shouldn't be used in the all_valid() method (we want all statuses there).
